src/state/opamRepositoryState.ml

```diff
@@ -127,13 +127,13 @@ module Cache = struct
 
 end
 
-let load_repo_opams repo =
-  let t = OpamConsole.timer () in
+let load_opams_from_dir repo_name repo_root =
+  (* FIXME: why is this different from OpamPackage.list ? *)
```

Comment:
Just a bunch of several modification from https://github.com/ocaml/opam/commit/423bb84b76e5fa5e04110b3e1ea7018617f1eb69 to https://github.com/ocaml/opam/commit/eebaa69e4a9f9aae77d07ad5b698c6487e40e619 i'd say...